README.md

```diff
@@ -0,0 +1,9 @@
+DocumentViewer
+=
+
+DocumentViewer is a viewer supporting PDF and DJVU files. 
+Many features are present, making it easier to open and view  files such as, 
+loading a document from file dialog or zooming in and out of a document.
+
+
+![DocumentViewer opened](https://raw.githubusercontent.com/Nova281/MyFiles/master/ed03ce06-27e9-11e6-8107-ff06f1c32923.png)
```

Comment:
You mustn't link to an image in your forked repo. You may decide to delete that repo one day and we have a broken link. Instead, add a "screenshot.png" file alongside your Readme.md and point to that.

Also, a big empty window isn't a nice screenshot. Open some PDF with it, make the window as small as visually pleasingly possible and use that. The version that's installable from HaikuDepot works for me.

Reply:
What build are you using?

Reply:
Haiku hrev51643 32bit.

Reply:
No wonder, I have been using 51602 ;(

Reply:
I’ll try that build and try running Document Viewer once more. Thanks!